.github/workflows/npm_publish.yml

```diff
@@ -0,0 +1,22 @@
+name: npm_publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.15.0
+      - run: npm install
+      - run: npm run build
```

Comment:
When I published, I also ran `npm init --scope=nosto` to publish it to the nosto organisation.